src/child/enums/actions.js

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
I'm not sure if this might be better in a _constants_ directory, rather than _enums_?

Just curious, but I noticed new blank lines at the top of some of these files - not sure why?